lib/features/amount_presets/models/user_presets.dart

```diff
@@ -1,8 +1,34 @@
 import 'package:equatable/equatable.dart';
 import 'package:givt_app/features/amount_presets/models/preset.dart';
 
+class AmountPresets extends Equatable {
```

Comment:
would be better to move it in a separate file amount_presets.dart?

Reply:
Yes, you're right!